index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" >
+</head>
+<body>
+    <main id="main">
+  
+        <h1 id="title">Dr. Norman Borlaug</h1>
+
+         <div id="img-div">
+
+            <img id="image" src="https://images.pexels.com/photos/15746942/pexels-photo-15746942.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="">
+              <div id="img-caption">
+                THis is a nice image caption
+              </div>
+
+         </div>
+
+         <div id="tribute-info"> 
+            <p> Here is the time line of Dr.Norman Borlaug</p>
+            <ul>
+            <li>1993: born</li>
+            <li>2007: graduated</li>
+            <li>2010: married</li>
+            </ul>
+
+           <p> LInk to Wikipedia article  <a id="tribute-link" href="#" target="_blank"> Wikipedia Entry</a>  </p> 
+         </div>
+
```

Comment:
I noticed that the header content is currently wrapped in a <div> with the id "img-div". Since the header represents a distinct section of the page, it would be more semantically appropriate to use a <section> tag instead. This helps convey the purpose and structure of the content more clearly.

Consider updating the HTML structure as follows:

`<section id="section-name">
  <!-- section content here -->
</section>`

By using the <section> tag, it enhances the readability and semantics of the code. 👍